example/demo.py

```diff
@@ -485,6 +485,28 @@ def img_click_callback(event):
     Y2 = np.random.randint(0, 20, 20)
     viz.dual_axis_lines(X, Y1, Y2)
 
+    # Example for Latex Support
+    viz.line(
+        X=[1, 2, 3, 4],
+        Y=[1, 4, 9, 16],
+        win="latex_support",
+        name=r'$\alpha_{1c} = 352 \pm 11 \text{ km s}^{-1}$',
+        update='append',
+        opts=dict(
+        showlegend=True,
+        title="Demo Latex in Visdom",
+        xlabel=r'$\sqrt{(n_\text{c}(t|{T_\text{early}}))}$',
+        ylabel=r'$d, r \text{ (solar radius)}$',
+        )
```

Comment:
Use `{'key': value}` formatting for dicts.

Reply:
oops.. I missed it again, I will make it proper.

Reply:
Changes Done...:)